README.md

```diff
@@ -11,26 +11,59 @@ More info available at: https://github.com/Cray-HPE/hms-trs-app-api
 1. Install the new cray-hms-trs-operator
 2. Create a TRSWorker using the config file [testapp1-http-v1.yaml](deploy/crds/test_crs/testapp1-http-v1.yaml)
 
-```bash
-kubectl apply -n services -f testapp1-http-v1.yaml
-kubectl -n services get TRSWorker
-```
+    ```bash
+    kubectl apply -n services -f testapp1-http-v1.yaml
+    kubectl -n services get TRSWorker
+    ```
 
-3. Check that the TRS pods were created.
+3. Check that the TRS Worker pods were created.
 
-```bash
-kubectl -n services get pods | grep trs
-```
+    ```bash
+    kubectl -n services get pods | grep trs
+    ```
+
+4. Check the logs of the TRS Worker pods for errors.
+
+    ```bash
+    kubectl -n services get pods | grep trsworker | awk '{print $1}' | xargs -I{} \
+    sh -c 'kubectl logs -n services -c trsworker {}; echo "-----------------"'
+    ```
+
+5. Check the logs of TRS Operator for errors.
+
+    ```bash
+    kubectl -n operators get pods | grep trs-operator | awk '{print $1}' | xargs -I{} \
+    sh -c 'kubectl logs -n operators {}; echo "-----------------"'
+    ```
 
-4. Check the logs of cray-hms-trs-operator
+6. When done remove the TRSWorker.
+
+    ```bash
+    kubectl delete -n services -f testapp1-http-v1.yaml
+    ```
+
+## Generate Source Code Using operator-sdk
+
+This project (hms-trs-operator) contains generated code. The code is generated by a tool called `operator-sdk` (see [operator-sdk](https://sdk.operatorframework.io/)).
+
+The version of `operator-sdk` used by hms-trs-operator is wrapped in the dockerfile [here](operator-sdk/Dockerfile). This docker image is built by the github workflows. It can be used to generate the source code and it should be manually run anytime the source code is changed.
+
+To generate the source code, run the following command from the root of the hms-trs-operator git source.
```

Comment:
Could you elaborate on the process of getting updated CRDs  (like the new Kafka CRDs) into the right locations so the right code can be generated?